Initium-Odp/src/com/universeprojects/miniup/server/ODPDBAccess.java

```diff
@@ -4491,12 +4493,46 @@ public String doCharacterAttemptAttack(ODPAuthenticator auth, CachedEntity user,
         Double charDex = getCharacterDexterity(sourceCharacter);
         Double monsterDex = getCharacterDexterity(targetCharacter);
         Random rnd = new Random();
+        
+        String status = "";
+        
+        Integer autoRunThreshold = (Integer) targetCharacter.getProperty("autoRunThreshold");
+        if(autoRunThreshold == null) autoRunThreshold = 0;
+        Integer autoRunChance = (Integer) targetCharacter.getProperty("autoRunChance");
+        if(autoRunChance == null) autoRunChance = 0;
+        
```

Comment:
Only run this logic if both `autoRunThreshold > 0 && autoRunChance > 0`. Spare some CPU cycles.